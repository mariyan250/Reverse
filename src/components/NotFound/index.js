import React from 'react';
import './index.scss';

const NotFound = () => {
  return (
    <div className="not-found vh-100 d-flex justify-content-center align-items-center">
      <h1>Not Found!</h1>
    </div >
  );
};

export default NotFound;