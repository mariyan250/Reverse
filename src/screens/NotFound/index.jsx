// React and style
import React from 'react';
import './index.scss';

// TODO translate
function NotFound() {
  return (
    <main className="not-found vh-100 d-flex justify-content-center align-items-center">
      <h1>Not Found!</h1>
    </main>
  );
}

export default NotFound;
