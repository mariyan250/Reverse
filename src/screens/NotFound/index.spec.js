import React from 'react';
import { render } from 'tests/test-utils';
import NotFound from './index';

describe('NotFound', () => {
  let initialState;

  beforeAll(() => {
    initialState = {};
  });

  describe('Rendering', () => {
    it('should render the component with initial state', () => {
      const { container } = render(<NotFound />, initialState);
      expect(container).toMatchSnapshot();
    });
  });
});
