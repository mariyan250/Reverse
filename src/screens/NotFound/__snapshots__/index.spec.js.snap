// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound Rendering should render the component with initial state 1`] = `
<div>
  <main
    class="not-found vh-100 d-flex justify-content-center align-items-center"
  >
    <h1>
      Not Found!
    </h1>
  </main>
</div>
`;
